daffodil-test/src/test/resources/org/apache/daffodil/section02/validation_errors/Validation.tdml

```diff
@@ -1902,7 +1902,7 @@
         
       <!-- Validation error for branch1:e1  -->
       <tdml:error>Validation Error</tdml:error>
-      <tdml:error>nonNegA failed facet checks due to: facet minInclusive (30)</tdml:error>
+      <tdml:error>e1 failed facet checks due to: facet minInclusive (30)</tdml:error>
```

Comment:
This is unexpected, any idea what cause this change in message?

Reply:
I just noticed that this test expects `<tdml:errors>`. Does it even make sense to expect validation errors if we don't even get an infoset? I wonder if this was just previously ignored since the tests didn't expect success? Should the expected validationErrors just be removed here, and in general tdml:validationErrors + tdml:errors in the same test should maybe be an TDMLException? We can open a bug for that if that makes sense.

Reply:
Good point, I did just add validationErrors to checkDiagnostics, which explains why the test just now failed.

I think it makes sense to make validationErrors errors in the same test a TDMLException in a future PR

Reply:
For the time being, I'll remeove the expected validation error